packages/shared/components/InformationTooltip.svelte

```diff
@@ -1,19 +1,20 @@
 <script lang="ts">
     import { FontWeight, Text, Tooltip } from 'shared/components'
     import { Position } from './enums'
-    import { localize } from '@core/i18n'
 
     export let anchor: HTMLElement
     export let position: Position = Position.Right
+    export let title: string
+    export let body: string
 </script>
 
 <Tooltip {anchor} {position}>
```

Comment:
I would add a restprops here just in case
```suggestion
<Tooltip {anchor} {position} {..$$restProps}>
```